packages/material-ui/src/ButtonBase/TouchRipple.js

```diff
@@ -280,8 +302,6 @@ TouchRipple.propTypes = {
   className: PropTypes.string,
 };
 
-TouchRipple.defaultProps = {
-  center: false,
-};
-
-export default withStyles(styles, { flip: false, name: 'MuiTouchRipple' })(TouchRipple);
+export default withStyles(styles, { flip: false, name: 'MuiTouchRipple' })(
+  withMuiName(React.memo(TouchRipple)),
```

Comment:
Should memo be used first, before withStyles?

Reply:
Current behavior is closer to previous behavior.